src/main/scala/org/broadinstitute/dsde/firecloud/utils/TSVFormatter.scala

```diff
@@ -68,14 +65,27 @@ object TSVFormatter {
   }
 
   /**
-    * JsValues are double-quoted. Need to remove them before putting them into a cell position
+    * Given an Attribute, creates a string that is safe to output into a TSV as a cell value.
+    * - if the input attribute contains a tab character, then double-quote it
     *
-    * @param value The JsValue to remove leading and trailing quotes from.
-    * @return Trimmed string value
+    * @param value The input attribute to make safe
+    * @return the safe value
     */
-  def cleanValue(value: JsValue): String = {
-    val regex = "^\"|\"$".r
-    regex.replaceAllIn(value.toString(), "")
+  def tsvSafeAttribute(attribute: Attribute): String = tsvSafeString(AttributeStringifier(attribute))
```

Comment:
Do we no longer need to trim the quotes?

Reply:
We explicitly need to NOT trim the quotes. If we trimmed quotes, then it would prevent any values-containing-a-tab from being quoted in the final TSV.

Elsewhere in the PR, we switch from using `value.toString()` to `AttributeStringifier` to turn a `JsValue` into a string. That change prevents `JsString` objects from being double-quoted … even though that wasn't actually a problem.